Chapters/4-1-Chapter23.md

```diff
@@ -0,0 +1,213 @@
+
+She looked at me, speechless, in a sort of way she reserved for no one else. As if to remind me that she was still my nather, that a good kaln would not take her patience for granted. But we both knew the truth. She was my nather, but my mother raised me alone.
+
+I raised the microphone on the ziyu and spoke.
+
+"Liffa. I don't know if you can hear this. I want you to know, that you are not forgotten. I'm trying to figure out where you'll land so I can get to you. If you can hear this... please send some sort of signal."
+
+The silence and static in my listener grew but my finger lingered on the speaking lever, waiting for a moment of hope. It didn't come.
+
+"That's enough," Setre said. "They'll find us here if you keep that up."
+
+"So let them! We can fight!" I responded. "What good is sitting?!"
+
+She looked at me in a way she reserved for no one else. As if to remind me that she was still my nather, that a good kaln would not take the patience of hae nather for granted.
+
+"Lin. What did your mother teach you to do with a problem?"
+
+"Don't talk to me about my mother."
+
+She paused and looked to the side, softened her voice. "She meant a lot to more than just you, you know."
+
+Still I couldn't look her in the eye, but I felt a small pang of guilt.
+
+"Solve it," I answered her.
+
+"Before that."
+
+"...Understand it."
+
+"And how do you understand a problem you've never seen before?"
+
+I caught her meaning, but late.
+
+Her voice had calmed to a cajole, "By talking to the people around you. We'll never get anywhere fighting, so let's look ahead and try our best."
+
+"You're just looking to shirk your responsibilities. Be honest, you didn't grieve when thi died."
+
+She raised her voice again, "I can't afford to grieve!" It was broken, and it was hurt, and I regretted what I had just said.
+
+With a loud thud of her spear against the ground, she lifted herself and stormed off toward the crowd, leaving me to my own thoughts. I looked out into the active crowd, mostly of Heikoi refugees. They were huddled around small fires dotting a wide stone basin where the eternal flame used to dance. A few adults were treating the wounded near a larger fire, making use of its bright light and sanitizing heat. Their spirit reminded me of Kipnu. When I woke from death that night on the beach of Ulukoma and the first thing I saw was his face and the unwavering concern for life. These were truly strangers in this place. They were displaced and came to us out of necessity. And yet the looks on their faces told me that they still felt uncomfortable around a people they were taught to distrust.
+
+"Setre, we need to talk," said Pomri, approaching her. I kept my position by the tree and listened closely.
+
+"Pomri," she spoke her name with a solemn pity, empathizing about Melna's loss, trying to keep her voice from breaking again. But Pomri was focused on her people.
```

Comment:
> empathizing about Melna's loss

general: show don't tell.
specific: expressing empathy is a collective set of actions, feelings, knowing, or being, and not really a singular action itself.

So I think you can choose a more specific word or description. For example: "she spoke her name with solemn pity, accounting for the weight of Melna's death."